api/starknet_api_openrpc.json

```
@@ -967,9 +967,8 @@
             "FELT": {
                 "type": "string",
                 "title": "Field element",
-                "$comment": "A field element, represented as a string of hex digits",
-                "description": "A field element. Represented as 4 leading zero bits followed by 63 hex digits",
-                "pattern": "^0x0[a-fA-F0-9]{63}$"
+                "description": "A field element. represented by at most 63 hex digits",
+                "pattern": "$^0x(0|[a-fA-F1-9]{1}[a-fA-F0-9]{0,62})$"
```

```suggestion
                "pattern": "^0x(0|[a-fA-F1-9]{1}[a-fA-F0-9]{0,62})$"
```

I think the `$` is here by mistake